FinalXmL/FinalXmL/Countrydata.cs

```diff
@@ -8,7 +8,7 @@
 
 
 
-namespace QuickType
+namespace QuickTypeCountry
```

Comment:
Rather than using the generic QuickType name I decided to change it to QuickTypeCountry

Reply:
Avoid putting commentary/justification/discussion in source code.  That becomes clutter.  Here in the pull request is a good place to have a conversation.
Also, as long as you're renaming the namespace, I'd drop QuickType all together.